tests/AssetManagerTest/View/Helper/AssetTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace AssetManagerTest\View\Helper;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use AssetManager\View\Helper\Asset;
+use Zend\ServiceManager\ServiceManager;
+
+class AssetTest extends TestCase
+{
+    public function testInvoke()
+    {
+        /* TODO need help
```

Comment:
https://github.com/RWOverdijk/AssetManager/blob/master/tests/AssetManagerTest/Service/AssetManagerTest.php#L45,L58

Here you can see it constructs its own MimeResolver, and then mocks a resolver. Does this help?